instat/dlgExportDataset.Designer.vb

```diff
@@ -24,23 +24,23 @@ Partial Class dlgExportDataset
     Private Sub InitializeComponent()
         Me.ucrBase = New instat.ucrButtons()
         Me.grpSaveOptions = New System.Windows.Forms.GroupBox()
+        Me.ucrChkOptions = New instat.ucrCheck()
         Me.grpOptions = New System.Windows.Forms.GroupBox()
-        Me.txtCols = New System.Windows.Forms.TextBox()
-        Me.txtRows = New System.Windows.Forms.TextBox()
+        Me.ucrChkUseRowNames = New instat.ucrCheck()
+        Me.ucrChkUseColumnNames = New instat.ucrCheck()
         Me.lblCols = New System.Windows.Forms.Label()
         Me.lblRows = New System.Windows.Forms.Label()
-        Me.chkRowNames = New System.Windows.Forms.CheckBox()
-        Me.chkColNames = New System.Windows.Forms.CheckBox()
-        Me.txtSheetName = New System.Windows.Forms.TextBox()
         Me.lblSheetName = New System.Windows.Forms.Label()
-        Me.txtAuthor = New System.Windows.Forms.TextBox()
         Me.lblAuthor = New System.Windows.Forms.Label()
-        Me.chkOptions = New System.Windows.Forms.CheckBox()
         Me.cmdBrowse = New System.Windows.Forms.Button()
-        Me.txtExportFile = New System.Windows.Forms.TextBox()
         Me.lblExport = New System.Windows.Forms.Label()
         Me.grpSheetSelection = New System.Windows.Forms.GroupBox()
         Me.ucrAvailableSheets = New instat.ucrDataFrame()
+        Me.ucrInputExportFile = New instat.ucrInputTextBox()
+        Me.ucrInputAuthor = New instat.ucrInputTextBox()
+        Me.ucrInputSheetName = New instat.ucrInputTextBox()
+        Me.ucrInputRows = New instat.ucrInputTextBox()
+        Me.ucrInputCols = New instat.ucrInputTextBox()
         Me.grpSaveOptions.SuspendLayout()
         Me.grpOptions.SuspendLayout()
         Me.grpSheetSelection.SuspendLayout()

```

Comment:
I don't think you need to have the data frame selector in a groupbox since it is only one item. You may need to edit the label of the selector appropriately.